src/options.cpp

```diff
@@ -1320,6 +1322,31 @@ void show_options(bool ingame)
             g->init_ui();
             if( ingame ) {
                 g->refresh_all();
+
+                DebugLog( D_INFO, DC_ALL ) << "Loaded tileset: " << OPTIONS["TILES"].getValue();
+
+                tile_loading_report(termap, "Terrain", "");
+                tile_loading_report(furnmap, "Furniture", "");
+                tile_loading_report(Item_factory::Item_factory().get_all_itypes(), "Items", "");
+                tile_loading_report(MonsterGenerator::generator().get_all_mtypes(), "Monsters", "");
```

Comment:
You might want to change `MonsterGenerator::get_all_mtypes` to return a const reference instead of a copy, but this copy is currently done only once, so it's not that bad.